website/source/api/system/replication-dr.html.md

```diff
@@ -0,0 +1,307 @@
+---
+layout: "api"
+page_title: "/sys/replication - HTTP API"
+sidebar_current: "docs-http-system-replication-dr"
+description: |-
+  The '/sys/replication/dr' endpoint focuses on managing general operations in Vault Enterprise replication sets
+---
+
+# `/sys/replication/dr`
+
+~> **Enterprise Only** – These endpoints require Vault Enterprise.
+
+## Check DR Status
+
+This endpoint print information about the status of replication (mode,
+sync progress, etc).
+
+This is an authenticated endpoint.
+
+| Method   | Path                         | Produces               |
+| :------- | :--------------------------- | :--------------------- |
+| `GET`    | `/sys/replication/dr/status`    | `200 application/json` |
+
+### Sample Request
+
+```
+$ curl \
+    https://vault.rocks/v1/sys/replication/dr/status
+```
+
+### Sample Response
+
+The printed status of the replication environment. As an example, for a
+primary, it will look something like:
+
+```json
+{
+  "mode": "dr-primary",
+  "cluster_id": "d4095d41-3aee-8791-c421-9bc7f88f7c3e",
+  "known_secondaries": [],
+  "last_wal": 0,
+  "merkle_root": "c3260c4c682ff2d6eb3c8bfd877134b3cec022d1",
+  "request_id": "009ea98c-06cd-6dc3-74f2-c4904b22e535",
+  "lease_id": "",
+  "renewable": false,
+  "lease_duration": 0,
+  "data": {
+    "cluster_id": "d4095d41-3aee-8791-c421-9bc7f88f7c3e",
+    "known_secondaries": [],
+    "last_wal": 0,
+    "merkle_root": "c3260c4c682ff2d6eb3c8bfd877134b3cec022d1",
+    "mode": "primary"
+  },
+  "wrap_info": null,
+  "warnings": null,
+  "auth": null
+}
+```
+
+## Enable DR Primary Replication
+
+This endpoint enables dr replication in primary mode. This is used when dr replication
+is currently disabled on the cluster (if the cluster is already a secondary, it
+must be promoted).
+
+| Method   | Path                         | Produces               |
+| :------- | :--------------------------- | :--------------------- |
+| `POST`   | `/sys/replication/dr/primary/enable` | `204 (empty body)` |
+
+### Parameters
+
+- `primary_cluster_addr` `(string: "")` – Specifies the cluster address that the
```

Comment:
I'd make it clear that this is optional, and if set blank, or not set at all, it will autodetect.